config/blog-core.php

```diff
@@ -16,4 +16,6 @@
     'api_url_with_path' => env('BLOG_API_URL', 'https://cslant.com') . '/' . env('BLOG_API_PATH', 'api/blog'),
 
     'laravel_domain' => env('LARAVEL_BLOG_DOMAIN', 'no-accept-domain'),
+
+    'expiration_view_time' => env('EXPIRATION_VIEW_TIME'),
```

Comment:
@TonyDuong0509  add the default minute in here. and need remove in the location to use this variable